app/notifications/process_notifications.py

```diff
@@ -162,6 +162,8 @@ def send_notification_to_queue(notification, research_mode, queue=None, recipien
 
             if recipient_id_type != IdentifierType.VA_PROFILE_ID.value:
                 tasks.insert(0, lookup_va_profile_id.si(notification.id).set(queue=QueueNames.LOOKUP_VA_PROFILE_ID))
+                tasks.insert(1, send_va_onsite_notification_task.si(notification.template.id, onsite_enabled)
```

Comment:
is "si" correct?  should it be "is"?

Reply:
I believe 'si' is short for 'signature' in celery.

Reply:
This is celery-specific stuff.